rules/scripting-user-agents.data

```diff
@@ -46,3 +46,6 @@ winhttp.winhttprequest
 # https://kong.github.io/unirest-java/
 # User-Agent: unirest-java/VERSION
 unirest-java/
+# http library
+# https://index.scala-lang.org/scalaj/scalaj-http
+# User-Agent: scalaj-http/2.4.2
```

Comment:
You probably meant to actually add the user agent string to match here?

Reply:
@theseion 
I have made the changes. Please have a look.

Reply:
The `#` at the beginning of the line starts a comment. Your user agent string will never be detected because it is commented out. On line 48 you have an example of how your next line should look. e.g.:
```
scalaj-http/
```

Reply:
You can revert your change to say `VERSION`, so that it is in line with line 47.

Reply:
@theseion 
I have made the changes. Please have a look.